.github/workflows/tests.yml

```diff
@@ -76,6 +76,7 @@ jobs:
           # - {device: "default.qubit", analytic: false, shots: 1000}
           # - {device: "default.qubit.tf", analytic: true, shots: 1000}
           - {device: "default.qubit.autograd", analytic: true, shots: 1000}
+          - {device: "default.mixed", analytic: true, shots: 1000}

```

Comment:
Nice! Do we also need to update the changelog?

Reply:
yes! Definitely